src/layouts/CommunityLayout.astro

```diff
@@ -16,55 +15,79 @@ interface Props {
 }
 
 // Max number of sketches we will show on this page
-const sketchLimit = 10;
+const sketchLimit = 8;
+// Max number of libraries we will show on this page
+const librariesLimit = 8;
+// Max number of libraries we will show on this page
+const eventsLimit = 9;
 
 const { sketches, libraries, pastEvents } = Astro.props;
 const currentLocale = getCurrentLocale(Astro.url.pathname);
 const t = await useTranslations(currentLocale);
 ---
 
 <BaseLayout title="Community">
-  <h2>{t("Sketches")}</h2>
-  {
-    sketches.slice(0, sketchLimit).map((sk) => (
-      <li>
-        {
-          // This isn't alt text ideal but we have no way of knowing what
-          // the thumbnail looks like, and the OpenProcessing API
-          // doesn't include that information
-        }
-        <Image
-          src={makeThumbnailUrl(sk.visualID)}
-          alt="Screenshot of sketch"
-          width={200}
-          height={200}
-        />
-        <a href={`/sketches/${sk.visualID}`}>{sk.title}</a>
-      </li>
-    ))
-  }
-  <a href="/sketches">See Sketches Archive</a>
-
-  <h2>{t("Libraries")}</h2>
-  {
-    libraries.map((entry) => (
-      <li>
-        <a href={getLibraryLink(entry)}>{removeLocaleAndExtension(entry.id)}</a>
-      </li>
-    ))
-  }
-
-  <a href="/libraries">See Libraries Archive</a>
-
-  <h2>{t("Events")}</h2>
-  {
-    pastEvents.map((entry) => (
-      <li>
-        <a href={`/past-events/${removeLocaleAndExtension(entry.id)}`}>
-          {removeLocaleAndExtension(entry.id)}
-        </a>
-      </li>
-    ))
-  }
-  <a href="/past-events">See Past Events Archive</a>
+  <section class="mt-md mb-xl">
+    <h2 class="mb-lg">{t("Sketches")}</h2>
+    <ul class="grid grid-cols-2 gap-y-xl gap-x-md md:grid-cols-4">
+      {
+        sketches.slice(0, sketchLimit).map((sk, i) => (
+          <li
+            class={(() => {
+              if (i === 0)
+                return "col-span-2 row-span-2 md:col-span-3 md:row-span-3";
+              if (i === 1) return "md:row-span-3";
+              if (i === 2) return "col-span-2 row-span-2";
+              if (i === librariesLimit - 1) return "row-span-2 col-span-2";
+              else return "md:row-span-2";
```

Comment:
I don't know if this is a priority but his is a situation where using `nth-child` style selectors may be a little more friendly.

Reply:
Yeah, I want to come back through and refactor these later. I think using an inline style tag in the layout would be good for stuff like this.